bedrock/firefox/templates/firefox/whatsnew/whatsnew-fx68-e.html

```diff
@@ -17,7 +17,6 @@ <h2>{{ _('Including Firefox Monitor, your watchful eye for data breaches') }}</h
     {% block monitor_button %}
       {{ monitor_button(
         entrypoint='mozilla.org-whatsnew68-e',
```

Comment:
Just a comment to point out that if this was a first commit, we would want these entrypoints to be standard. In this case the experiment is almost over; no change requested.

Reply:
This PR updates the global macros that all pages use, so these changes will be enforced everywhere that uses those macros, no exceptions. If that's an issue, then I suggest this PR doesn't merge until the experiment is over?

Reply:
It looks like the experiment was stopped in https://github.com/mozilla/bedrock/pull/7588, so I think we're good to merge this as-is.